modules/nodes-cluster-overcommit-configure-masters.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-cluster-overcommit.adoc
+
+[id='nodes-cluster-overcommit-configure-masters_{context}']
+= Configuring masters for overcommitment
```

Comment:
need rewrite, if even possible, in 4.0

Reply:
@sjenning 
>>need rewrite, if even possible, in 4.0

Are you suggesting that overcommit is not in 4.0?
Who would know for certain?

Reply:
let's not document this for now